Description: Enable tapping even when a physical left button is available.
Bug-Ubuntu: https://launchpad.net/bugs/320585

Index: xserver-xorg-input-synaptics-1.8.2/src/synaptics.c
===================================================================
--- xserver-xorg-input-synaptics-1.8.2.orig/src/synaptics.c
+++ xserver-xorg-input-synaptics-1.8.2/src/synaptics.c
@@ -622,10 +622,10 @@ set_default_parameters(InputInfoPtr pInf
     palmMinWidth = priv->minw + range * (10.0 / 16);
     emulateTwoFingerMinW = priv->minw + range * (7.0 / 16);
 
-    /* Enable tap if we don't have a phys left button */
-    tapButton1 = priv->has_left ? 0 : 1;
-    tapButton2 = priv->has_left ? 0 : 3;
-    tapButton3 = priv->has_left ? 0 : 2;
+    /* Enable tap */
+    tapButton1 = 1;
+    tapButton2 = 3;
+    tapButton3 = 2;
 
     /* Enable multifinger-click if only have one physical button,
        otherwise clickFinger is always button 1. */
